ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/LedgerApiTestTool.scala

```diff
@@ -122,17 +129,27 @@ object LedgerApiTestTool {
         sys.exit(1)
       })
 
+    val included_p =
```

Comment:
What's a `p`? I would suggest spelling it out (and using `camelCase` instead of `snake_case`).